app/SendAmount.tsx

```diff
@@ -41,31 +44,48 @@ export default function SendAmount() {
   const { isRequest } = params;
 
   const onPressSend = async () => {
-    dispatch(setIsUserLoading(true));
     if (!chosenContact?.phoneNumbers) return;
 
     // phone number for recipient in send flow and 'purchaser' (aka requestee) in request flow
     const phoneNumber = chosenContact?.phoneNumbers[0].number;
 
     if (!phoneNumber) return;
+    if (currentUser?.accountBalance <= 0) {
+      throw new Error('not enough');
+    }
+
+    if (amount > currentUser?.accountBalance) {
+      dispatch(setModalText('Not enough funds'));
+      dispatch(setIsModalVisible(true));
+      return;
+    }
+    dispatch(setIsUserLoading(true));
```

Comment:
This functionality is looking good ! thanks 👍

Reply:
I have updated the code, check it out